src/widget.ts

```diff
@@ -179,255 +185,272 @@ export class WidgetOptions {
  * 
  *   cyberusKeyButton.create('cyberus-key-widget-container');
  * });
- * 
+ *
  * If fullOpenIdLogin is true, then a user will be redirected to cyberuskey.com to process the login. This is the recommended configuration. If it’s false, the login process is done only on your side and you will have to e.g. handle errors.
- * 
+ *
  * ```
  *
  * @export
  * @class CyberusKeyWidget
  */
 export class CyberusKeyWidget {
-  private _serverUrl: URL;
-  private _theme: string;
-  private _clientId: string;
-  private _redirectUri: string;
-  private _scope: OpenIdScopeParser;
-  private _geoProvider: GeoProvider
-  private _state: string;
-  private _nonce: string;
-  private _initialized: boolean;
-  private _inProgress: boolean;
-  private _animation: WidgetAnimation;
-  private _containerElement: Element;
-  private _responseType: string;
-  private _origin: string;
-  private _autoplay: boolean;
-  private _fullOpenIdLogin: boolean;
-
-  /**
-   * Creates an instance of CyberusKeyWidget.
-   * 
-   * @param {WidgetOptions} options
-   * @memberof CyberusKeyWidget
-   */
-  constructor(options: WidgetOptions) {
-    if (!options || !options.clientId || !options.redirectUri) {
-      throw new Error('CyberusKeyWidget: Missing options(s).')
+    private _serverUrl: URL;
+    private _theme: string;
+    private _clientId: string;
+    private _redirectUri: string;
+    private _scope: OpenIdScopeParser;
+    private _geoProvider: GeoProvider
+    private _state: string;
+    private _nonce: string;
+    private _initialized: boolean;
+    private _inProgress: boolean;
+    private _animation: WidgetAnimation;
+    private _containerElement: Element;
+    private _responseType: string;
+    private _origin: string;
+    private _autoplay: boolean;
+    private _fullOpenIdLogin: boolean;
+
+    /**
+     * Creates an instance of CyberusKeyWidget.
+     *
+     * @param {WidgetOptions} options
+     * @memberof CyberusKeyWidget
+     */
+    constructor(options: WidgetOptions) {
+        if (!options || !options.clientId || !options.redirectUri) {
+            throw new Error('CyberusKeyWidget: Missing options(s).')
+        }
+
+        let theme = options.theme || 'default';
+        const serverUrl = options.serverUrl || 'https://production-api.cyberuskey.com';
```

Comment:
we can change now all refetences from ''https://production-api.cyberuskey.com'' to https://api.cyberuskey.com